src/main/java/com/emc/object/s3/request/DeleteBucketRequest.java

```diff
@@ -0,0 +1,38 @@
+package com.emc.object.s3.request;
+
+import com.emc.object.Method;
+import com.emc.object.util.RestUtil;
+
+import java.util.List;
+import java.util.Map;
+
+public class DeleteBucketRequest extends GenericBucketRequest {
+
+    // if this bucket contains objects/versions
+    private boolean backgroundDeletion;
```

Comment:
Recommend changing the name here to something more meaningful - something like `recursivelyDeleteAllData`.